src/main/resources/templates/users/login.ftl

```diff
@@ -0,0 +1,35 @@
+<html>
+<head>
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css">
```

Comment:
this dependencies have to be exctracted from header to some separate partial
no need to do it right now
just leave a ToDo comment here